src/ukify/ukify.py

```diff
@@ -1420,6 +1420,7 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         '--devicetree',
         metavar = 'PATH',
         type = pathlib.Path,
+        action = 'append',
```

Comment:
this is a backwards-incompatible change. Maybe that's acceptable, IDK - up to other reviewers

Reply:
There isn't much choice if we want to allow providing multiple DTBs without a config file. Why would anyone want to rely on the current behavior anyway? Isn't this supposed to be an internal to ukify.py implementation detail internal to ukify.py?

Reply:
I'll consider this resolved as there is now a separate `--devicetree-auto` option